lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,20 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
+ * @function
+ * @public
+ *
+ * @description
+ * This function returns undefined if there was no last flash.
```

Comment:
"if no flash has been performed yet" or ""if no flash has been started yet" might be more readable than "if there was no last flash"?